includes/notifications/notifications-send.php

```diff
@@ -0,0 +1,165 @@
+	private Event_Repository_Interface $event_repository;
+
+	/**
+	 * Notifications_Send constructor.
+	 *
+	 * @param Event_Repository_Interface $event_repository    Event repository.
+	 * @param Attendee_Repository        $attendee_repository Attendee repository.
+	 */
+	public function __construct(
+		Event_Repository_Interface $event_repository,
+		Attendee_Repository $attendee_repository
+	) {
+		$this->event_repository    = $event_repository;
+		$this->attendee_repository = $attendee_repository;
+		add_action( 'wporg_gp_translation_events_email_notifications_1h', array( $this, 'send_notifications' ), 10, 1 );
+		add_action( 'wporg_gp_translation_events_email_notifications_24h', array( $this, 'send_notifications' ), 10, 1 );
+	}
+
+	/**
+	 * Send notifications to the attendees of the event.
+	 *
+	 * @param int $post_id Post ID.
+	 *
+	 * @return void
+	 */
+	public function send_notifications( int $post_id ) {
+		$event     = $this->event_repository->get_event( $post_id );
+		$attendees = $this->attendee_repository->get_attendees( $event->id() );
+		foreach ( $attendees as $attendee ) {
+			$this->send_email_notification( $event, $attendee );
+		}
+	}
+
+	/**
+	 * Send an email notification to the attendee of the event.
+	 *
+	 * @param Event    $event        The event.
+	 * @param Attendee $attendee     The attendee.
+	 *
+	 * @return void
+	 */
+	public function send_email_notification( Event $event, Attendee $attendee ): void {
+		$user    = get_user_by( 'ID', $attendee->user_id() );
+		$subject = $this->get_email_subject( $event );
+		$message = $this->get_email_message( $user, $event );
+		wp_mail(
+			$user->user_email,
+			$subject,
+			$message,
+			'Content-Type: text/html'
+		);
+	}
+
+	/**
+	 * Get the email subject.
+	 *
+	 * @param Event $event The event.
+	 *
+	 * @return string
+	 */
+	private function get_email_subject( Event $event ): string {
+		$subject  = esc_html__( 'Event Reminder.', 'gp-translation-events' );
+		$subject .= ' ';
+		$subject .= sprintf(
+		// translators: %1$s: Event title. %2$s Time until event starts.
+			esc_html__( 'You have the %1$s event in %2$s', 'gp-translation-events' ),
+			esc_html( $event->title() ),
+			esc_html( $this->calculate_time_until_event( $event->start() ) )
+		);
+
+		return $subject;
+	}
+
+	/**
+	 * Get the email message.
+	 *
+	 * @param WP_User $user  The user.
+	 * @param Event   $event The event.
+	 *
+	 * @return string
+	 */
+	private function get_email_message( WP_User $user, Event $event ): string {
+		$start_date = $event->start();
+		// translators: %s: User display name.
+		$message  = sprintf( esc_html__( 'Hi %s', 'gp-translation-events' ), $user->display_name );
+		$message .= '<br><br>';
+		$message .= sprintf(
+			// translators: %1$s: Event title. %2$s: Time until event starts.
+			esc_html__( 'You have the %1$s event in %2$s.', 'gp-translation-events' ),
+			esc_html( $event->title() ),
+			esc_html( $this->calculate_time_until_event( $event->start() ) )
+		);
```

Comment:
```suggestion
		$message .= esc_html( sprintf(
			// translators: %s: Time until event starts.
			__( 'The event will start in %s.', 'gp-translation-events' ),
			$this->calculate_time_until_event( $event->start() )
		) );
```
			esc_html__( 'You have the %1$s event in %2$s.', 'gp-translation-events' ),
			esc_html( $event->title() ),
			esc_html( $this->calculate_time_until_event( $event->start() ) )
		);
```